tasks/install.yml

```diff
@@ -0,0 +1,65 @@
+---
+- name: download latest dokuwiki
+  get_url:
+    url: https://download.dokuwiki.org/src/dokuwiki/dokuwiki-stable.tgz
+    dest: /tmp/dokuwiki-stable.tgz
+ 
+- name: unarchive dokuwiki-stable.tgz to {{ dokuwiki_install_path }}
+  unarchive:
+    src: /tmp/dokuwiki-stable.tgz
+    dest: "{{ dokuwiki_install_path | dirname }}"
+    owner: www-data
+    remote_src: yes
+    list_files: yes
+  register: file_list
+
+- name: remove stupid version name from folder path
+  file:
+    src: "{{ dokuwiki_install_path | dirname }}/{{ file_list['files'][0] }}"
+    dest: "{{ dokuwiki_install_path }}"
+    owner: www-data
+    state: link
+
+- name: Install xml-php package used by dokuwiki.
+  apt:
+    name: "{{ item }}"
+  with_items:
+    - php-ldap
+    - php-xml
+  when: ansible_pkg_mgr == "apt"
+
+- name: Copy Logo to Server
+  copy:
+    src: files/logo.png
+    dest: "{{dokuwiki_install_path}}/data/media/wiki/logo.png"
+    owner: www-data
+
+- name: Remove default plugins
+  file:
+    path: '{{dokuwiki_install_path}}/lib/plugins/{{ item }}'
+    state: absent
+  with_items:
+    - authpdo
+    - authmysql
+    - authpgsql
+    - authplain
+    - authad
+
+- name: set acls
+  copy:
+    src: 'files/acl.auth.php'
+    dest: '{{dokuwiki_install_path}}/conf/acl.auth.php'
```

Comment:
Is this file even respected if you set `dokuwiki_useacl` to 0 like it is in the defaults?

Reply:
no it isnt

Reply:
now its only copyed when useacl ist set

Reply:
Best practice would be to template it.